components/system/components/fragments/TableComponents.js

```diff
@@ -100,7 +100,6 @@ const STYLES_COLUMN = css`
   align-items: flex-start;
   justify-content: space-between;
   align-self: stretch;
-  min-width: 10%;
```

Comment:
`min-width: 10%` is a style added to catch edge cases on safari and mobile safari, did you test in those browsers?

Reply:
Ahh good to know. No worries, I can leave it in there!